com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDSubTarget.cs

```diff
@@ -139,22 +139,23 @@ protected SubShaderDescriptor PostProcessSubShader(SubShaderDescriptor subShader
                 passDescriptor.requiredFields.Add(subShaderField);
 
                 IncludeCollection finalIncludes = new IncludeCollection();
-                var includeList = passDescriptor.includes.Select(include => include.descriptor).ToList();
 
                 // Replace include placeholders if necessary:
                 foreach (var include in passDescriptor.includes)
                 {
-                    if (include.descriptor.value == CoreIncludes.kPassPlaceholder)
-                        include.descriptor.value = subShaderInclude;
-                    if (include.descriptor.value == CoreIncludes.kPostDecalsPlaceholder)
-                        include.descriptor.value = postDecalsInclude;
-                    if (include.descriptor.value == CoreIncludes.kRaytracingPlaceholder)
-                        include.descriptor.value = raytracingInclude;
-                    if (include.descriptor.value == CoreIncludes.kPathtracingPlaceholder)
-                        include.descriptor.value = pathtracingInclude;
-
-                    if (!String.IsNullOrEmpty(include.descriptor.value))
-                        finalIncludes.Add(include.descriptor.value, include.descriptor.location, include.fieldConditions);
+                    var path = include.path;
```

Comment:
This seems like an error in that it used to be modifying the values stored in the passDescriptor (which may be static global)..   now it should not modify the existing structure...